pkg/packagekit/updates.jsx

```diff
@@ -506,7 +506,7 @@ class RestartServices extends React.Component {
         if (this.props.tracerRunning) {
             body = (
                 <Flex spaceItems={{ default: 'spaceItemsSm' }} alignItems={{ default: 'alignItemsCenter' }}>
-                    <Spinner isSvg size="sm" />
+                    <Spinner size="sm" />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18672-20230426-085951-0a013b1d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)